easybuild/easyconfigs/y/yank/yank-0.25.2-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'PythonBundle'
+
+name = 'yank'
```

Comment:
It seems like `YANK` is the official software name, so we should use that here (and rename the easyconfig file and the directory it's in accordingly)